src/models/aspera-desktop.model.ts

```diff
@@ -31,8 +33,10 @@ class DesktopGlobals {
   desktopVerified = false;
   /** The unique ID for the website */
   appId: string;
-  /** Map of dropzones created by querySelector */
-  dropzonesCreated: Map<string, {event: string; callback: (event: any) => void}[]> = new Map();
+  /** The session ID for the current user */
+  sessionId: string;
```

Comment:
Since `sessionId` can really be optional (is opt-in by the client), does TS support setting optional member fields? Like `sessionId?: string`?

Reply:
Yeah, it does support that, forgot to add the `?` to it